### PR TITLE
Added Youtube Trending Feed Scrapper

### LINKED ISSUE #752: Scrap YouTube Trending Section
# Details
Come up with a script that stores the title of the video, youtube channel name, number of likes, dislikes, views, no of comments, date of publishing, description of the video (if possible) in a suitable database.

Create another script to fetch that database. Refer to this script for database fetching and displaying:  [Tweet Fetcher](https://github.com/avinashkranjan/Amazing-Python-Scripts/tree/master/Tweet-Fetcher)

------------------------------------------
Updates: Database chosen- MongoDB

### PR DESCRIPTION
**Pull request by XZANATOL**
# Description

Added ``Youtube Trending Feed Scrapper`` with the ability to save using ``MongoDB`` and ``CSV File``

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [x] Yes
- [ ] No

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] Documentation Update

## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.

### CODE REVIEW

**Comment by XZANATOL** [comment]
I think Lint doesn't know how to deal with defined global variables. I've re-ran the script right now and it generated no errors/warnings.

Error of the action can be found on line 80 of ``Lint Code Base`` job:
``Youtube Trending Feed Scrapper/youtube_scrapper.py:105:9: E0601: Using variable 'df' before assignment (used-before-assignment)``

File: Youtube Trending Feed Scrapper/youtube_scrapper.py [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+                views = views,
+                date = date)
+    # Save record
+    record.save()
+
+
+def append_to_df(section, title, channel, link, views, date):
+    """Appends a record to dataframe."""
+    global df
+    df = df.append({"section": section,
```
Comment (kaustubhgupta) [inline Youtube Trending Feed Scrapper/youtube_scrapper.py:105]: this is a very common error while deploying data science applications in the cloud. I have faced this issue many times. The solution to this define this "df" as empty data frame at the beginning of the script

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

File: Youtube Trending Feed Scrapper/youtube_scrapper.py (hunk 1 above)
Comment (XZANATOL) [reply Youtube Trending Feed Scrapper/youtube_scrapper.py:105]: Hmmm, If I did that then an empty DataFrame will be initialized with every call of this function causing the discard of any previous data that is stored in it. :thinking:

Comment (XZANATOL) [reply Youtube Trending Feed Scrapper/youtube_scrapper.py:105]: or do you mean defining it at the beginning of the file?

Comment (kaustubhgupta) [reply Youtube Trending Feed Scrapper/youtube_scrapper.py:105]: At the beginning of the file

Comment (XZANATOL) [reply Youtube Trending Feed Scrapper/youtube_scrapper.py:105]: hmm ok, lemme edit that.

**Comment by XZANATOL** [comment]
haha nice

**Review by kaustubhgupta — APPROVED** [review]
💯

**Review by Kushal997-das — APPROVED** [review]
LGTM 💯

**Review by antrikshmisri — APPROVED** [review]
LGTM 👍🏻

**Review by santushtisharma10 — APPROVED** [review]

**Review by avinashkranjan — APPROVED** [review]
LGTM..Great Job @XZANATOL

**Pull request merged**